### ISSUE TITLE
[Handbook] Update the Repository Overview, Projects, & Mentorship Programs

### ISSUE THREAD

**Issue by Anita-ihuman** [layer5io/layer5#2153]
**Current Behavior**
These sections are missing some content from the newly updated Handbook draft.

**Desired Behavior**
Update the Repository Overview, Projects & Mentorship Programs section with the details from the [new draft](https://docs.google.com/document/d/1vxc8MLhDetIlBa1DCUTMbYLGWL4Df4K1xB_Ov5oUK7o/edit#heading=h.vtdtai5j3tq7) of the handbook.

---
**Screenshots / Mockups**

**Alternatives**

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

**Comment by khanjasir90** [comment]
@Anita-ihuman can i work on this!

**Comment by debo19** [comment]
Go ahead @khanjasir90 :rocket:

**Comment by Shatakshi0805** [comment]
Can I work on this?

**Comment by khanjasir90** [comment]
@Shatakshi0805 we can work together on this issue if you don't mind!

**Comment by khanjasir90** [comment]
@leecalcote @warunicorn19 @Anita-ihuman since "Community Bridge Program was not included in New Handbook" so should I delete all the files related to it or there is any other way that I am not aware of?

**Comment by Anita-ihuman** [comment]
I think that was an omission,
I think it should be included there as well.

**Comment by warunicorn19** [comment]
@leecalcote do we need to include community bridge? or would just LFX do?

**Comment by warunicorn19** [comment]
guys any updates on this?

**Comment by khanjasir90** [comment]
umm..... working! would make a PR soon.

**Comment by Aniket1312** [comment]
hey can I also join here.

**Comment by khanjasir90** [comment]
@Aniket1312  you can go ahead ! i got a bit busy i'll join you.

**Comment by moazzam07** [comment]
Can I also work on it

**Comment by Aniket1312** [comment]
Ya sure

**Comment by moazzam07** [comment]
PR created please go through it.

**Comment by warunicorn19** [comment]
@YashKamboj are you going to take a look at this issue?

**Comment by YashKamboj** [comment]
> @YashKamboj are you going to take a look at this issue?
> 

Yeah ,if noone is working on it .

**Comment by YashKamboj** [comment]
Hey @warunicorn19 , can we talk so i can know what exactly changes i have to make, if you are free.

**Comment by frankiefab100** [comment]
Has this issue been resolved?

**Comment by YashKamboj** [comment]
> Has this issue been resolved?

Yes, will be merged today ,i think made all the changes they requested.
